dbt_subprojects/dex/models/aggregator_trades/dex_aggregator_trades.sql

```diff
@@ -44,8 +43,8 @@ SELECT
     , token_pair
     , token_bought_amount
     , token_sold_amount
-    , token_bought_amount_raw
-    , token_sold_amount_raw
+    , try_cast(token_bought_amount_raw as uint256) as token_bought_amount_raw
+    , try_cast(token_sold_amount_raw as uint256) as token_sold_amount_raw
```

Comment:
all set with casting as `uint256 `@jeff-dude

Reply:
okay, i am checking with other teams where this data is coming from to be safe. this is a good safeguard plan if we don't have any definitive answers to solve at project level spells

Reply:
i also noticed another dune teammate request [this](https://github.com/duneanalytics/spellbook/pull/6325/files), looks like same request but on `dex.trades`. that one uses `cast`, but we could convert to `try_cast` there too